Makefile

```diff
@@ -0,0 +1,56 @@
+artifact_name       := limited-partnerships-web
+version             := "unversioned"
+
+.PHONY: all
+all: build
+
+.PHONY: clean
+clean:
+	rm -f ./$(artifact_name)-*.zip
+	rm -rf ./build-*
+	rm -rf ./dist
+	rm -f ./build.log
+
+.PHONY: build
+build:
+	npm ci
+	npm run build
+
+# .PHONY: lint
```

Comment:
Thoughts on this?

I copied the OE Web Makefile and made changes, then commented out the targets that aren't yet applicable (until we complete the other tasks).

Should I leave them commented out (easy to add back in later/soon) or just delete for now?

Reply:
you can uncomment them if they will be needed

Reply:
They will be needed but they won't work until those other tasks are delivered, so I'd essentially be delivering a Makefile with non-working targets. I think I'd prefer to leave them commented out with a general qualifying comment.

Reply:
Done.